### PR TITLE
Questions card styling

### PR DESCRIPTION
**Pull request by djavanm**
Questions route should be styled completely now.

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -16,18 +16,18 @@
 @import "Question";
 @import "StudentQuestionContainer";
 
-body {background: grey; font-family: $font}
+body {
+  background: #d4d6d9;
```
Comment (Myrdden) [inline src/styles/main.scss]: If you want a white background then just make it white

Comment (Myrdden) [inline src/styles/main.scss:20]: If you want a white background then make sure you update all the other components that have white text. Also, I think just a plain white would look best.

File: src/styles/main.scss [hunk 2]
```diff
@@ -16,18 +16,18 @@
...
+  font-family: $font
+}
 
 .main-container {
   color: white;
-  position: fixed;
-  top: $nav-height; bottom: 0;
-  width: 100vw;
+  width: 97vw;
```
Comment (Myrdden) [inline src/styles/main.scss:26]: Why?

File: src/styles/main.scss [hunk 3]
```diff
@@ -16,18 +16,18 @@
...
+  height: 100vh;
 }
 
 .side-margins {
-  position: absolute;
-  top: 0; bottom: 0;
-  left: 20%; right: 20%;
+   padding: 0% 15% 2% 17%;
```
Comment (Myrdden) [inline src/styles/main.scss:31]: Also why?

File: src/styles/main.scss (hunk 1 above)
Comment (Myrdden) [reply src/styles/main.scss:20]: I can't say I'm a huge fan of the card background being the same as the page background. Could we make the default card some shade of grey or keep the blue and just make a lighter blue hover state? It doesn't really matter what I just feel like they should be different.

File: src/styles/main.scss (hunk 3 above)
Comment (djavanm) [reply src/styles/main.scss:31]: We don't want to utilize position absolute unless necessary. The container is positioning the cards in the same manner with the one padding rule.

Comment (Myrdden) [reply src/styles/main.scss:31]: Might want to use $nav-height in there somewhere in case that changes.

**Pull request merged**